<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>791</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>SAP系统：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxSap">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>表名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtTableName">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>分隔符：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtDelimeter">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>读取数量：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinCount">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRun">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>执行</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="btnAddField">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>0</y>
               <width>61</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>增加字段</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnDeleField">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>0</y>
               <width>51</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>删除字段</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnSwitchSelectField">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>0</y>
               <width>51</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>反选</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnSwitchHeader">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>0</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>切换抬头</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnReadFields">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>61</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>读取字段</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tblFields">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <column>
             <property name="text">
              <string>选择</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>字段清单</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>描述</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="btnAddCon">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>61</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string> 增加条件</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnDelCon">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>0</y>
               <width>61</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>删除条件</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnSwitchSelectCon">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>0</y>
               <width>61</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>反选</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tblConditions">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="autoScrollMargin">
             <number>0</number>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>选择</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>条件</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabTables">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
